tasks/compile.ts

```diff
@@ -4,17 +4,12 @@ var Builder = require('systemjs-builder');
 var util = require('gulp-util');
 var ts = require('gulp-typescript');
 
+var tsProject = ts.createProject('tsconfig.json');
+
 export const compile = (gulp, config) => {
   gulp.task('compile:main-server', () => {
-    return gulp.src('src/**/*.ts')
-      .pipe(ts({
-        noImplicitAny: false,
-        typescript: require('typescript'),
-        module: 'commonjs',
-        emitDecoratorMetadata: true,
-        experimentalDecorators: true,
-        moduleResolution: 'node'
-      }))
+    return tsProject.src()
```

Comment:
it's important to use `tsProject.src()` here since we need to also include the type definitions. By using `tsProject.src()` we're reverting control over the files to our `tsconfig.json` to specify our entry/type definition files rather than managing it in our gulpfile